CHANGELOG/v1.6.0.md

```diff
@@ -0,0 +1,294 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.25.x -> v1.28.x
+- Workload Cluster: v1.23.x -> v1.28.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Highlights
+
+* MachinePools are now supported in ClusterClass Clusters
+* Metrics, profiling and other diagnostics are now served securely by default
+* Types in `ipam.cluster.x-k8s.io` have graduated to `v1beta1`
+
+## Deprecation Warning
+
+- The API version `v1alpha4` is no longer served in this release.
+  - [Version migration guide](https://main.cluster-api.sigs.k8s.io/developer/providers/version-migration.html)
+  - [GitHub issue #8038](https://github.com/kubernetes-sigs/cluster-api/issues/8038)
+  - [API deprecation details](https://main.cluster-api.sigs.k8s.io/contributing#removal-of-v1alpha3--v1alpha4-apiversions)
+- The API version `v1alpha3` has been completely removed in this release.
+- Flag `--metrics-bind-addr` is [deprecated](https://github.com/kubernetes-sigs/cluster-api/pull/9264) for all controllers
+
+## Changes since v1.5.0
+## :chart_with_upwards_trend: Overview
+- 388 new commits merged
+- 6 breaking changes :warning:
+- 15 feature additions ✨
+- 35 bugs fixed 🐛
+
+## :memo: Proposals
+- Community meeting: Add proposal for karpenter integration feature group (#9571)
+
+## :warning: Breaking Changes
+- API: Remove v1alpha3 API Version (#8997)
+- API: Stop serving v1alpha4 API Versions (#8996)
+- clusterctl: Improve Context handling in clusterctl (#8939)
+- Dependency: Bump to controller-runtime v0.16 (#8999)
+- Metrics/Logging: Implement secure diagnostics (metrics, pprof, log level changes) (#9264)
+- util: : Remove go-vcs dependency from releaselink tool (#9288)
```

Comment:
```suggestion
- util: Remove go-vcs dependency from releaselink tool (#9288)
```

Reply:
Is this actually a breaking change? 

Also from a user perspective this is not relevant at all. I think I would move it to others

Reply:
@sbueringer makes sense, I agree it is not directly relevant to CAPI users. Should we retitle the merged PR accordingly and move it after?

Reply:
Sounds good to me

(I forgot to mention: this is explicitly called out in the provider migration doc, so I think it's fine to not mention it as breaking here)

Reply:
@sbueringer it has been moved to "Others" sections, can you please help us moving the same PR to that section in beta/rc.(0/1) releases notes in GitHub UI?

Reply:
Done